src/plugins/intel_cpu/src/emitters/README.md

```diff
@@ -0,0 +1,116 @@
+# CPU plugin JIT emitters
+
+## Abstract
+CPU plugin JIT emitters implement element-wise operations. The key features are:
+ * implementation is based on just-in-time (JIT) code generation,
+ * input and output data is stored in vector or scalar registers.
+ 
+ Both features allows to use JIT emitters as pre or post-operations, fuse several emitters in one operation or in compilers approach frameworks.
+ 
+For just in time source code generation `Xbyak JIT Assembler` is used:
+ * [Xbyak for X64](https://github.com/herumi/xbyak)
+ * [Xbyak for ARM64](https://github.com/fujitsu/xbyak_aarch64)
+
+
+Emitter eimplementations are placed here: https://github.com/openvinotoolkit/openvino/tree/master/src/plugins/intel_cpu/src/emitters.  
+
+There are two types of emitters:
+ * [plugin emitters](https://github.com/openvinotoolkit/openvino/tree/master/src/plugins/intel_cpu/src/emitters/plugin)
+ * [snippets emitters](https://github.com/openvinotoolkit/openvino/tree/master/src/plugins/intel_cpu/src/emitters/snippets)
+
+Each emitter is linked with with OpenVINO operation:
+ * plugin emitter is linked in JIT kernel `jit_uni_eltwise_generic::create_eltwise_emitter` method which implementation depends on platform:
+   * [X64 specific](https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_cpu/src/nodes/eltwise.cpp)
+   * [ARM64 SIMD specific](https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_cpu/src/nodes/kernels/aarch64/jit_uni_eltwise_generic.cpp)
+ * snippet emitter is linked in [CPUTargetMachine](https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_cpu/src/emitters/snippets/x64/cpu_generator.cpp)
```

Comment:
Let remove for now. Doesn't introduce much sense now. Lets cover Snippets topic in details separatelly.

Reply:
done